sidebars.js

```diff
@@ -288,6 +288,7 @@ const sidebars = {
             'guides/data-binding/how-to-bind-to-a-collection',
             'guides/data-binding/binding-from-code',
             'guides/data-binding/how-to-create-a-custom-data-binding-converter',
+            'guides/data-binding/how-to-bind-multiple-properties',
```

Comment:
Since you added this page, doe ru and zh pages still work?

Reply:
I overlooked this, incorrectly believing that sidebars had localized versions. There must be a built-in fallback, because the localized versions still work.